alfiere.py

```diff
@@ -0,0 +1,70 @@
+from Pezzo import Pezzo
+
+
+class alfiere(Pezzo):
+    """
+    implementa l'alfiere
+    """
+
+    def __init__(self, colore, posizione=None):
+        super().__init__(colore, posizione, 'alfiere')
+        #
+        self.graphic_rep = '\u2657' if self.colore == 'W' else '\u265d'
+
+    def verifica_mossa(self, destinazione):
+        """
+                verifica se l'alfiere può essere mosso alla destinazione
+
+                Parameters
+                ----------
+                destinazione : coppia di coordinate (list)
+                    posizione di destinazione della mossa
+
+                Returns
+                -------
+                bool
+                indica se la mossa è legale o no
+
+             """
+        # se tutte le condizioni precedenti sono verificati allora entra e verifia anche queste n
```

Comment:
Volendole mantenere, queste line di codice dovrebbero essere spostate dentro il primo if (riga 35), cioè dopo i controlli generali
In realtà poi non le usate quasi mai (solo alle righe 41 e 42), per cui le eliminarei e userei sempre lo stesso  metodo per riferimi alle coordinate